build/tasks/unit.js

```diff
@@ -29,7 +30,7 @@ module.exports = function ( grunt ) {
 						output
 					);
 
-					if ( !noop ) {
```

Comment:
Were we never setting noop, then?

Reply:
Yes, we were and still are. On line 20. A test case's first value is the input, and the optional second value is the expected output after flipping. If no second value is provided (which we do in quite a few cases), then we simply assert the value is not affected by flipping (e.g. non-flippable properties and things like `@noflip`). In those cases, no need for a roundtrip given it would be the same assertion.

By default, we always roundtrip if the test isn't `noop`. But this commit introduces the ability to disable `roundtrip`.